app/views/admin/visit_requests/_emails.slim

```diff
@@ -0,0 +1,6 @@
+.twelve.wide.column
+  = send_email_to_recipient_ids_link(visitors_ids)
+  = send_email_to_attendees_link(confirmed_ids)
+  = send_email_to_applied_users_link(applied_ids) if event.confirmation?
```

Comment:
I would move these `if's` to helper methods like we have in other places.

Reply:
@floor114  please, re-check